harness/types/sdks.ts

```diff
@@ -1,7 +1,8 @@
 export const Sdks = {
     nodejs: 'NodeJS',
     dotnet: 'DotNet',
+    go: 'Go'
     // TODO add once python is ready
```

Comment:
update comment to say once Java is ready, too? since we're only running the Test Harness with the Java sdk in local mode right now right?

Reply:
yup, will add it in 😄